content/learn/video.html

```diff
@@ -0,0 +1,199 @@
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+        </div>
+        <div class="paragraph">
+            <p>A KIE Drop is a short, focused, five minute (or less) video used to promote an idea, one of our KIE
+                projects, or some feature in a project. Do you have an idea for a KIE Drop? Get in contact with us!</p>
+        </div>
+    </div>
+</div>
+<div class="sect1">
+    <h2 id="_kieweek">KIE Week 2021</h2>
+    <div class="sectionbody">
+        <div class="justify-content-center text-center">
+            <iframe width="560" height="315"
+                src="https://www.youtube.com/embed/videoseries?list=PLo3ZScdD9hW6UtFRbgOfYbdrcPkIjatQr"
+                title="YouTube video player" frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+        </div>
+        <div class="paragraph">
+            <p>KIE Week series -- Exploring Process & Business Automation -- where we’ll take you on a general technical
+                and cultural journey through approaches to integrating process and business automation rules. You’ll
+                delve into workflow through the eyes of some of our core international Red Hat® engineers’ community
+                KIE/Drools projects. And, we’ll round out the week with a roundtable discussion about real-world
+                customer implementations of the Red Hat products built from KIE projects.
+                <br />
+                Our KIE Week will offer a single one-hour session each day. Join that one that especially resonates with
+                you or register for them all! </p>
+        </div>
+    </div>
+</div>
+<div class="sect1">
+    <h2 id="_droolsdmnengine">Drools DMN Engine</h2>
+    <div class="sectionbody">
+        <div class="justify-content-center text-center">
+            <iframe width="560" height="315"
+                src="https://www.youtube.com/embed/videoseries?list=PLdbdefeRIj9RW_bWDmx17oXzKshu-4T9H"
+                title="YouTube video player" frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+        </div>
+        <div class="paragraph">
+            <p>A collection of presentations covering Drools DMN open source engine</p>
+        </div>
+    </div>
+</div>
+<div class="sect1">
+    <h2 id="_make_your_own_videos">Make your own videos</h2>
+    <div class="sectionbody">
+        <div class="paragraph">
+            <p>If you published a public video featuring Drools, <a href="../community/team.html">let us know</a>
+                and we'll happily re-share it on our KIE Community channels.</p>
+        </div>
+    </div>
+</div>
+<h2 id="_make_your_own_videos">Video archive</h2>
```

Comment:
```suggestion
<h2 id="_make_your_own_videos">Make Your Own Videos: The Archive</h2>
```

Reply:
this one meant to be really video archive section; nice catch tho